contributing/DEVELOPING.md

```diff
@@ -47,6 +47,7 @@ The Quick Start Guide's  [One-time setup](getting-started-quick.md#one-time-setu
 | `gulp lint`                                                             | Validates against Google Closure Linter.                              |
 | `gulp lint --watch`                                                     | Watches for changes in files, Validates against Google Closure Linter.|
 | `gulp lint --fix`                                                       | Fixes simple lint warnings/errors automatically.                      |
+| `gulp lint --files=path/to/file.js`                                     | Lints specified files.                                                |
```

Comment:
This should be `gulp lint --files=<files-path-glob>`

Reply:
And `Lints just the files provided.`

Reply:
Whoops, fixed :)

Reply:
The PR description is out of date. Please update it before merging.